bazel/deps.bzl

```diff
@@ -21,3 +17,12 @@ def deps(repo_mapping = {}):
         build_file = "@com_github_3rdparty_bazel_rules_curl//:BUILD.bazel",
         repo_mapping = repo_mapping,
     )
+
+    maybe(
+        git_repository,
+        name = "boringssl",
+        commit = "fc44652a42b396e1645d5e72aba053349992136a",
+        remote = "https://boringssl.googlesource.com/boringssl",
+        shallow_since = "1627579704 +0000",
+        repo_mapping = repo_mapping,
+    )
```

Comment:
Let's move this up above where openssl was so that it comes _before_ we pull down libcurl.